hyper/common/exceptions.py

```diff
@@ -71,3 +71,22 @@ class MissingCertFile(Exception):
     The certificate file could not be found.
     """
     pass
+
+
+# Create our own ConnectionError.
+try:  # pragma: no cover
+    ConnectionError = ConnectionError
+except NameError:  # pragma: no cover
+    class ConnectionError(Exception):
+        """
+        An error occurred during connection to a host.
+        """
```

Comment:
Why?

Here's the `ConnectionResetError` done the same way in master: https://github.com/Lukasa/hyper/blob/development/hyper/common/exceptions.py#L41